polkadot/xcm/xcm-runtime-apis/tests/mock.rs

```diff
@@ -144,7 +144,8 @@ parameter_types! {
 	pub const BaseXcmWeight: Weight = Weight::from_parts(100, 10); // Random value.
 	pub const MaxInstructions: u32 = 100;
 	pub const NativeTokenPerSecondPerByte: (AssetId, u128, u128) = (AssetId(HereLocation::get()), 1, 1);
-	pub UniversalLocation: InteriorLocation = [GlobalConsensus(NetworkId::Westend), Parachain(2000)].into();
+	pub UniversalLocation: InteriorLocation = [GlobalConsensus(NetworkId::ByGenesis(WESTEND_GENESIS_HASH)), Parachain(2000)].into();
+	pub static AdvertisedXcmVersion: XcmVersion = 4;
```

Comment:
Westend is not important here, and `AdvertisedXcmVersion` is unused

```suggestion
	pub UniversalLocation: InteriorLocation = [GlobalConsensus(NetworkId::ByGenesis([0; 32)), Parachain(2000)].into();
```